x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/overview.tsx

```diff
@@ -548,7 +548,6 @@ export const Overview: React.FC = () => {
                 {!indexPermissions && isOrganization && (
                   <EuiFlexItem>{documentPermissionsDisabled}</EuiFlexItem>
                 )}
-                {indexPermissions && <EuiFlexItem>{credentials}</EuiFlexItem>}
```

Comment:
question(non-blocking): Do we generally want to unit-test this kind of behavior, or is that too fine-grained?

Reply:
It certainly wouldn't hurt. In this case, the bug was a long-lived oversight that existed in the legacy code, which did not have unit tests. So even if we had written granular tests here, the issue would have still existed, unless someone saw it in the UI, like Ross did. 

Thanks for the quick review!